integrationtests/src/test/java/io/kroxylicious/proxy/BaseIT.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.DeleteTopicsResult;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.common.TopicCollection;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.kroxylicious.test.tester.KroxyliciousTester;
+import io.kroxylicious.testing.kafka.junit5ext.KafkaClusterExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(KafkaClusterExtension.class)
+public abstract class BaseIT {
+
+    protected static CreateTopicsResult createTopics(Admin admin, List<NewTopic> topics) {
+        try {
+            var created = admin.createTopics(topics);
+            assertEquals(topics.size(), created.values().size());
```

Comment:
we've both assertj and junit assertions flowering all over the project.  It would be nice to reach agreement and go one way or the other.  I have a preference for the former.

Reply:
+1 to everything you say @SamBarker

Reply:
Agreed on using assertJ. I've made this change in all the files this PR touches.